### DISCUSSION TITLE
File Version Preview URL

### DISCUSSION THREAD

**Discussion by DanTm99** [pnp/pnpcore#1409] [Q&A] [answered]
I'm able to get a preview URL for a file, for embedding in an iframe. E.g. Using `IFile.GetFilePreview().GetUrl`.

But is it possible to somehow embed a specific file version (i.e. `IFileVersion`) in an iframe?

**Comment by jansenbe** [comment, marked as answer]
See https://github.com/pnp/pnpcore/issues/1408.
